sdk/cosmosdb/cosmos/CHANGELOG.md

```diff
@@ -1,16 +1,118 @@
 # Release History
 
 ## 4.0.0 (2023-09-12)
+🎉 v4 release! 🎉 Many new features, bug fixes, and a few breaking changes.
+- Summary of new added features 
+  - Diagnostics: has been added to response objects of api operations ie. point lookups, bulk & batch operations, query and error responses.
+  - Hierarchical Partitioning: Containers with hierarchical partitions are now supported.
```

Comment:
can we add the documentation link for all these features?

Reply:
https://learn.microsoft.com/azure/cosmos-db/hierarchical-partition-keys

Reply:
Done